tests/frontend/test_kibana_app/cypress/integration/step-definitions/filters/the-user-add-a-new-filter.and.js

```diff
@@ -0,0 +1,24 @@
+import { And } from 'cypress-cucumber-preprocessor/steps';
+import { clickElement, fillSelectorField, elementIsVisible} from '../../utils/driver';
+
+import {
+  addFilterButton,
+  filterSuggestionList,
+  filterOperatorList,
+  filterParams,
+  saveFilterButton,
+  selectedOperator
+} from '../../pageobjects/filters/filters.page';
+ 
+
+And('The user add a new filter', () => {
+  elementIsVisible(addFilterButton);
+  clickElement(addFilterButton);
+  fillSelectorField(filterSuggestionList,'rule.level');
+  //fillSelectorField(filterOperatorList,'is');
+  elementIsVisible(filterOperatorList);
+  clickElement(filterOperatorList);
+  clickElement(selectedOperator);
+  fillSelectorField(filterParams,'7');
```

Comment:
This parameter is hardcoded. Can we make it parametrized?
we will always be validating our application with a '7'
I know how we sometimes test it manually this way, but maybe we can do it better.